### DISCUSSION TITLE
How can I keep battery at 80% when laptop is booting? How can I get MyAsus and G-Helper to work together?

### DISCUSSION THREAD

**Discussion by Kaito1877** [seerge/g-helper#3523] [Q&A]
I just got an Asus Rog Strix G18, uninstalled both Armoury Crate and MyAsus to keep g-helper but learnt that during the system booting, the g-helper wont keep the battery below 80% and continue on charging. I was told that MyAsus would work if I used it to limit my battery but I was wondering if I could use them both at the same time? What I had in mind was that I would close g-helper from start up tasks, make sure MyAsus limits the battery at %80 when the laptop starts, then stop Asus services by opening g-helper and use the 80% battery limit in g-helper since laptop is already booted. I would like to know if any conflict could happen between both by doing so, or if anybody could guide me on how to use both without conflicting them. I will be going to University next year and I feel like I should value the importance of battery life, that's why I am a bit worried about its well being.

**Comment by seerge** [comment]
@Kaito1877 hello,

When your device reboots it resets almost all of it's settings, including charge limit. So it will be charging till windows starts again and either Asus Optimization Service or G-Helper starts and sets a charge limit again. That's normal and how it works.

If you run G-Helper as admin at least once - it would create a separate startup task that runs and sets charge limit a bit earlier (before user login)
